keel-api/src/test/resources/examples/alb-example.yml

```diff
@@ -9,8 +9,8 @@ spec:
     stack: example
     detail: albec2v1
   locations:
-    accountName: test
-    vpcName: vpc0
+    account: test
```

Comment:
Accounts have two properties: name and id. is removing `Name` confusing? I always get confused when we just have `account` in various clouddriver code - sometimes it's name, sometimes it's id.

Reply:
In Keel's API we never use ids. I agree in CloudDriver where it's ambiguous. Keel's API should never expect users to know ids.

Reply:
Okay